patches/chrome-browser-browsing_data-chrome_browsing_data_remover_delegate.h.patch

```diff
@@ -13,8 +13,8 @@ index b8840d0c9ce032ce19738cc2b806161798245a01..992b6514053de0c816ed56d623d36c1c
 @@ -136,7 +137,8 @@ class ChromeBrowsingDataRemoverDelegate
      kAccountCompromisedCredentials = 39,
      kFaviconCacheExpiration = 40,
-     kSecurePaymentConfirmationInstruments = 41,
--    kMaxValue = kSecurePaymentConfirmationInstruments,
+     kSecurePaymentConfirmationCredentials = 41,
+-    kMaxValue = kSecurePaymentConfirmationCredentials,
 +    kIPFSCache = 42,
```

Comment:
`#define kMaxValue kIPFSCache = 42, kMaxValue = kIPFSCache,` ? Otherwise at least compress `kIPFSCache = 42, kMaxValue = kIPFSCache,` to one line

Reply:
Squashed into one line in ac103cc. `kMaxValue` is used in many headers, so it maybe safer not to redefine it.